src/main/resources/io/viash/runners/nextflow/channel/channelFromParams.nf

```diff
@@ -56,6 +56,12 @@ private List<Tuple2<String, Map<String, Object>>> _paramsToParamSets(Map params,
 
   /* combine arguments into channel */
   /**********************************/
+
+  // Fetch default values for arguments
+  def defaultArgs = config.allArguments
+    .findAll { it.containsKey("default") }
+    .collectEntries { [ it.plainName, it.default ] }
+
```

Comment:
Please add an entry to the changelog for this change. Ideally, it'd be a separate PR because it's not related to the removal of previously deprecated arguments ^^

Reply:
Added a CHANGELOG entry. I added this change because otherwise the tests were failing. So I would like to avoid a separate PR if possible.